### DISCUSSION TITLE
设置每篇文章的标签图标

### DISCUSSION THREAD

**Discussion by redhat123456** [volantis-x/hexo-theme-volantis#682] [Q&A] [answered]
我想知道我该如何实现这样的效果
![image](https://user-images.githubusercontent.com/57751257/139381308-526eef2b-87c2-4c80-ada8-17ec9469264d.png)
就是如何将分类的标签导出来

**Comment by xaoxuu** [comment, marked as answer]
看截图像是 stellar 主题的，代码在这： https://github.com/xaoxuu/hexo-theme-stellar/blob/5ab54f321d2670ad28f06be9691c587d1f30886a/layout/_partial/main/post_list/post_card.ejs#L64-L66
